app/views/playComponents/answer_row.scala.html

```diff
@@ -0,0 +1,57 @@
+@*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import viewmodels.AnswerRow
+
+@(row: AnswerRow)(implicit messages: Messages)
+
+@hiddenLabel = @{
+    row.label + ".hidden"
+}
+
+@rowAnswer = @{
+    if(row.answerIsMessageKey) {
+        messages(row.answer).capitalize
+    } else {
+        Html(row.answer)
+    }
+}
+
+<li>
+    <div class="cya-question">@Html(messages(row.label, row.labelArgs: _*))</div>
```

Comment:
this doesn't seem like the current implementation of check your answers https://design-system.service.gov.uk/patterns/check-answers/

Happy to discuss this in more detail if I have the wrong end of the stick